.devcontainer/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM --platform=$BUILDPLATFORM ubuntu:24.04
+ARG TARGETARCH
+
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get update && \
+    apt-get install -y \
+    adwaita-qt \
+    alsa-utils \
+    build-essential \
+    clangd \
+    gdb \
+    git \
+    libadwaitaqt1 \
+    pipewire \
+    pipewire-alsa \
+    pulseaudio-utils \
+    python3-pip \
+    qt6-wayland \
+    sudo \
+    udev \
+    vim && \
+    pip3 install pre-commit --break-system-packages
+
+COPY tools/debian_buildenv.sh /tmp/
+
+RUN /tmp/debian_buildenv.sh setup
+
+RUN userdel -f -r ubuntu && \
```

Comment:
What is the reason to for this change of user name? Is there an issue with the defaults?